app/src/main/java/com/onboarding/calculator/util/ConstantsUtils.java

```diff
@@ -20,4 +20,20 @@
     public static final int CORRECTION_FACTOR = 1;
     public static final double DEFAULT_RESULT = 0.0;
 
+    public static final String ZERO_STRING_TEST = "0";
```

Comment:
Do not add constants for the tests here. Add them in a new constant utils class in the tests package or in the test class

Reply:
Solved!